app/Http/Controllers/PublishedPostsController.php

```diff
@@ -4,16 +4,19 @@
 
 use App\Models\Post;
 use App\Support\Settings;
+use Illuminate\Http\Request;
 
 class PublishedPostsController extends Controller
 {
     /**
      * Handle the incoming request.
      */
-    public function __invoke(): \Illuminate\View\View
+    public function __invoke(Request $request): \Illuminate\View\View
```

Comment:
can you import the return type so it becomes `: View`

Reply:
Sure